### DISCUSSION TITLE
Correction in the linkerd documentation

### DISCUSSION THREAD

**Discussion by Chakki1301** [linkerd/linkerd2#9452] [General]
https://linkerd.io/2.12/tasks/using-ingress/#traefik-2x - Traefik 2.x ingressroute middleware should be in emojivoto namespace instead of traefik namespace. Thanks.

**Comment by mateiidavid** [comment]
@Chakki1301 would you be up for submitting a PR in our [website](https://github.com/linkerd/website) repository with the change?
